src/components/QuickGrid/QuickGrid.Props.ts

```diff
@@ -1,6 +1,11 @@
 import * as React from 'react';
 import { ITooltipProps } from '../Tooltip/Tooltip.props';
 
+export interface IQuickGrid {
```

Comment:
why is it showing this here? this was a part of some other pull request?

Reply:
yes i moved it in other file